---
title: Reconciling phylogenetic and population genomic data create robust species
  limits
#author: "Lara Wootton"
date: "12/04/2021"
output: pdf_document
---



Utility of next gen data for species delimitation

Species delimitation is widely recognised as a crucial component of systematic biology, with ramifications for many downstream analyses. The recent abundance and affordability of genome-wide, multi-locus datasets have provided unprecedented power to circumscribe robust species boundaries, even in the face of traditional difficulties to delimitation, such as few distinctive morphological characteristics [@Singhal2018; @Linck2019], speciation with gene flow [@Jackson2017; @Georges2018; @Chan2020] and hybrid speciation [@Quattrini2019]. Multi-locus datasets commonly used for species delimitation can be classified into two primary types. The first is generated via sequence capture approaches, where a probe set is designed to target specific loci, and typically consists of exons or other conserved genomic regions, together with their flanking regions. While they predominately designed to resolve deep-time relationships, their more variable flanking regions have been used in shallow time contexts [@fairfield, @lemmon]. The second commonly-used datatype is generated via restriction site associated DNA sequencing (RadSeq), or genotyping-by-sequencing approaches. In this case, the data is widely spread across the genome, and consists of both coding and non coding loci, making it useful for population level approaches, and more shallow timescales [@Mccormack2013; @refs]. 

In order to circumscribe and validate species boundaries, the focus to date has been on the development of numerous methodological approaches. These techniques vary widely, from population genetic clustering methods [@Pritchard2000; @Frichot2014] to phylogenetic methods [@astral], to species-tree based methods [@bpp, @Snapp]. Given a strong emphasis on using as many methods as possible to validate species boundaries [@Carstens2013], it is perhaps unsurprising that researchers have been pushing the limits as to what their datasets can do. For example, Radseq data are now routinely used for phylogenetic analysis. While this isn't problematic in of itself, Radseq data are potentially less informative than sequence capture if there are deep time splits [@ref], and the recovered relationships tend to be sensitive to missing data and bioinformatic processing decisions [@Leache2015]. Another recent trend is to call single nucleotide polymorphisms (SNPs) from sequence capture data in order to utilise population genetic methods [@Zarza2016; @Winker2018; @Chan2020; @Erikson2020]. However, as the resultant SNPs are retrieved from predominantly functional regions, they may not meet the central assumptions of Hardy-Weinberg equilibrium and linkage disequilibrium required by many population-level analyses without rigorous filtering [@Pritchard2000; @ref], in addition to potentially altering how the outputs should be interpreted [@Helyar2011; @Kirk2011; Silva2020].

A potential way around these problems is to use a 'two-pronged' approach, by incorporating both Radseq and sequence capture within a single study. This is becoming more common within the field of phylogenetics [@Manthey2016; @Obiol2021; @refs], but surprisingly has not yet been widely used for species delimitation [exceptions @Linck2020]. Possibly because the different resolutions of the data make them difficult to directly compare within a given framework. A major advantage in using both is to have independent evaluation of species boundaries from different genomic regions; corroboration between the two datatypes would be a powerful ratification of species limits. Approaching species delimitation the two different datatypes could also help resolve species boundaries where their lineages are in the 'grey zone' of speciation, which remain problematic to resolve [@refs]. The differing information content of Radseq and sequence capture would mean that the problem is essentially approached from both sides of the species continuum. Finally, 



The Cape grass genus _Ehrharta_ Thunb. represents an ideal model system within which to test whether speciation process varies with elevation in the GCFR. The genus currently consists of approximately 45 taxa (species and subspecies) and has a Gondwanan distribution, with species occurring in Australasia, New Zealand, Malesia, Reunion, Madagascar and Africa. However, a substantial proportion of _Ehrharta_ species (22 in total), are GCFR endemics, comprising of a core low-elevation clade that is subtended by a grade of mid- to high-elevation lineages [@GibbsRussell1987; @Verboom2003]. This low-elevation clade, which associates primarily with summer-arid renosterveld and succulent karoo scrub habitats, has undergone an adaptive radiation in response to the Late Miocene aridification, and has been interpreted as the primary radiation in _Ehrharta_ [@Verboom2003; @Verboom2004]. Nevertheless, although the mid- and high- elevation lineages of Cape _Ehrharta_ comprise just six species in total, four of these species contain multiple subspecies. In addition, field observations suggest that there may be more subtle morphological differentiation in the high-elevation clades than current taxonomy suggests, hinting at the existence of cryptic species. 

The most recent infrageneric taxonomy of the Cape _Ehrharta_ is that of @GibbsRussell1987, who divided the genus into seven informal species groups. Most of these groups occur predominately in the arid, lower-elevation, renosterveld and succulent karoo vegetation regions of the GCFR. Within these lowlands groups, there is little taxonomic confusion as the species are clearly morphologically distinct, with characters that easily distinguish the species. Conversely, within two of the three higher-elevation, fynbos-restricted groups, Setacea and Ramosa, the species limits are less clear. The Setacea group consists of two species, _E. setacea_ and _E. rupestris_, the former comprising four subspecies (_disticha, setacea, uniflora_ and _scabra_), while the latter contains three (_rupestris, tricostata_ and _dodii_). As noted by @GibbsRussell1987, these species and subspecies are morphologically similar and difficult to distinguish. While the subspecies in both groups tend to have distinct ranges, many of them occur in sympatry in the Caledon area, where intermediates also occur. Species limits are even less clear within the Ramosa group, which contains two species, _E. ramosa_ with two subspecies (_aphylla_ and _ramosa_), and _E. rehmannii_ with three subspecies (_rehmannii, filiformis_ and _subspicata_). @GibbsRussell1989 observe that "the five taxa comprising the Ramosa group intergrade and few consistently reliable characters adequately separate the species and subspecies". As these taxa were delimited using only morphological and anatomical characters, it is possible that the application of genetic data may bring clarity to species boundaries in this group.



In contrast, few studies have attempted to ratify species boundaries by integrating both sequence capture and Radseq datatypes [exceptions being @Linck]. Yet, under certain circumstances, using a 'two-pronged' approach could be strongly advantageous. For example, current methods still find delimiting recent species and lineages within the grey zone of speciation difficult, with sequence capture potentially not having good enough resolution, and Radseq data often leading to oversplitting. The differing information content of Radseq and sequence capture means that the problem is essentially approached from both sides of the species continuum, potentially leading to more robust species limits. 



Grey zone - 

Approaching species delimitation from two different timescales could help resolve species boundaries where their lineages are in the 'grey zone' of speciation. Ambiguous lineages  In addition, Radseq has a tendency to over split species with strong pop structure.Finally, by sequencing the same individuals in both account for intraspecific variability.  People have been trying to do this using SNPs from UCE, but this is just repackaging the same information rather than been different independent measures. Phylogenetics is starting to make use of both types to resolve shallow radiations, yet this trend has yet to catch on in species delimitation studies. Potential reasons for this is expense, and difficulty of reconciling different resolutions. 



The two datatypes have very different phylogenetic resolutions. Radseq assumed to be neutral, reflect population-level processes. Sequence capture in conserved and generally reflects older timescales (with the exception of flanking regions) Both data types therefore have different strengths and weaknesses for delimiting species [as covered extensively in @Harvey]. In brief:
Phylogenetics is starting to make use of both types to resolve shallow radiations, yet this trend has yet to catch on in species delimitation studies. Using both to incoporate both shallow and deep time scales may help resolve persistent delimitation problems, such as grey zone and strong pop structure.



The utility of the datasets depends on the system being studied. Sequence capture data is better at resolving  older timescales, although with flanking regions)



Benefits of intergrating both: Independent - better than trend of calling SNPs from UCE



