---
title: Reconciling phylogenetic and population genomic data create robust species
  limits
#author: "Lara Wootton"
date: "12/04/2021"
output: pdf_document
---



Utility of next gen data for species delimitation

Species delimitation is widely recognised as a crucial component of systematic biology, with ramifications for many downstream analyses. The recent abundance and affordability of genome-wide, multi-locus datasets have provided unprecedented power to circumscribe robust species boundaries, even in the face of traditional difficulties to delimitation, such as few distinctive morphological characteristics [@Singhal2018; @Linck2019], speciation with gene flow [@Jackson2017; @Georges2018; @Chan2020] and hybrid speciation [@Quattrini2019]. Multi-locus datasets commonly used for species delimitation can be classified into two primary types. The first is generated via sequence capture approaches, where a probe set is designed to target specific loci, and typically consists of exons or other conserved genomic regions, together with their flanking regions. While they predominately designed to resolve deep-time relationships, their more variable flanking regions have been used in shallow time contexts [@fairfield, @lemmon]. The second commonly-used datatype is generated via restriction site associated DNA sequencing (RadSeq), or genotyping-by-sequencing approaches. In this case, the data is widely spread across the genome, and consists of both coding and non coding loci, making it useful for population level approaches, and more shallow timescales. 

In order to circumscribe and validate species boundaries, the focus to date has been on developing and applying numerous methodological approaches [@Carstens2013]. The various methodologies has been extensively tested, and there has been substantial debate over the suitability of each. In contrast, the utility of using more than one multilocus datatype within a study has not been adequately explored. Where more than one genetic dataset has been used, it is typically a mtDNA or cpDNA data with a low number of genes that is then used to inform downstream analyses with Next-gen data.

Most species delimitation studies using multi-locus data will generate a single data type. Then, to validate hypothesised species boundaries, will focus on applying a variety of analytical methods, such as stacey, to that dataset. 

In contrast, few studies have attempted to ratify species boundaries by integrating both sequence capture and Radseq datatypes [exceptions being @Linck]. Yet, under certain circumstances, using a 'two-pronged' approach could be strongly advantageous. For example, current methods still find delimiting recent species and lineages within the grey zone of speciation difficult, with sequence capture potentially not having good enough resolution, and Radseq data often leading to oversplitting. The differing information content of Radseq and sequence capture means that the problem is essentially approached from both sides of the species continuum, potentially leading to more robust species limits. 

Approaching species delimitation from two different timescales could help resolve species boundaries where their lineages are in the 'grey zone' of speciation. Ambiguous lineages  In addition, Radseq has a tendency to over split species with strong pop structure.Finally, by sequencing the same individuals in both account for intraspecific variability.  People have been trying to do this using SNPs from UCE, but this is just repackaging the same information rather than been different independent measures. Phylogenetics is starting to make use of both types to resolve shallow radiations, yet this trend has yet to catch on in species delimitation studies. Potential reasons for this is expense, and difficulty of reconciling different resolutions. 



The two datatypes have very different phylogenetic resolutions. Radseq assumed to be neutral, reflect population-level processes. Sequence capture in conserved and generally reflects older timescales (with the exception of flanking regions) Both data types therefore have different strengths and weaknesses for delimiting species [as covered extensively in @Harvey]. In brief:
Phylogenetics is starting to make use of both types to resolve shallow radiations, yet this trend has yet to catch on in species delimitation studies. Using both to incoporate both shallow and deep time scales may help resolve persistent delimitation problems, such as grey zone and strong pop structure.



The utility of the datasets depends on the system being studied. Sequence capture data is better at resolving  older timescales, although with flanking regions)



Benefits of intergrating both: Independent - better than trend of calling SNPs from UCE



Oversplitting is a major

issue in RAD-Seq data sets and it disproportionately

affects species with higher natural levels of divergence among alleles (

To date, the focus has been on ratifying results using multiple methodological approaches rather 

What they are and their pros and cons. Methods typically used on them.

In a typical species delimitation study that uses genome-wide data, a single dataset will be created.

Problems with only using one type, and how phylogenetics is using both [@Manthey2016; @Obiol2021]. Yet to be properly done in species delimitation yet though.

In phylogentics, the both types of data are starting to be used in order to overcome their weaknesses.
Species continuum?

Yet despite the insights provided by these multilocus datasets there are still uncertainties. 

Species continuum concept - approaching in both