---
title: Reconciling phylogenetic and population genomic data create robust species
  limits
#author: "Lara Wootton"
date: "12/04/2021"
output: pdf_document
---



Species delimitation is widely recognised as a crucial component of systematic biology, with ramifications for many downstream analyses. The recent abundance and affordability of genome-wide, multi-locus datasets have provided unprecedented power to circumscribe robust species boundaries, even in the face of traditional difficulties to delimitation, such as few distinctive morphological characteristics [@Singhal2018; @Linck2019], speciation with gene flow [@Jackson2017; @Georges2018; @Chan2020] and hybrid speciation [@Quattrini2019]. Multi-locus datasets commonly used for species delimitation can be classified into two primary types. The first is generated via sequence capture approaches, where a probe set is designed to target specific loci, and typically consists of exons or other conserved genomic regions, together with their flanking regions. While they predominately designed to resolve deep-time relationships, their more variable flanking regions have been used in shallow time contexts [@fairfield, @lemmon]. The second commonly-used datatype is generated via restriction site associated DNA sequencing (RadSeq), or related genotyping-by-sequencing approaches. In this case, the data is widely spread across the genome, and consists of both coding and non-coding loci, making it useful for population level approaches, and more shallow timescales [@Mccormack2013; @refs]. 

Very few species delimitation studies have used both sequence capture and RadSeq to infer species boundaries [exceptions include @Linck; @Harvey]. Instead, the focus to date has been on the development of numerous methodological approaches. These techniques vary widely, from population genetic clustering methods [@Pritchard2000; @Frichot2014] to phylogenetic methods [@astral], to species-tree based methods [@bpp, @Snapp]. There is a strong emphasise within the integrative taxonomic framework to use as many of these methods as possible to validate species boundaries [@Carstens2013]. Therefore, it is perhaps unsurprising that researchers have been pushing the limits as to what their datasets can do, with little regard to the suitability of their genomic datasets for their chosen analyses. For example, Radseq data are now being used for phylogenetic analysis of older radiations, despite being potentially less informative than sequence capture if there are deep time splits [@ref], and the recovered relationships tend to be sensitive to missing data and bioinformatic processing decisions [@Leache2015]. Another recent trend is to call single nucleotide polymorphisms (SNPs) from sequence capture data in order to utilise population genetic methods [@Zarza2016; @Winker2018; @Chan2020; @Erikson2020]. However, as the resultant SNPs are retrieved from predominantly functional regions, they may not meet the central assumptions of Hardy-Weinberg equilibrium and linkage disequilibrium required by many population-level analyses without rigorous filtering [@Pritchard2000; @ref], in addition to potentially altering how the resulting outputs should be interpreted [@Helyar2011; @Kirk2011; Silva2020].

To alleviate this problem, we propose that there should be a shift in focus towards using multiple genomic datasets, rather than multiple methodologies, to validate species boundaries, as is becoming more common within the field of phylogenomics [@Manthey2016; @Obiol2021; @refs]. Incorporating both Radseq and sequence capture data within a single study has multiple advantages. Firstly, using both datatypes gives an independent evaluation of species boundaries from different genomic regions; corroboration between the two datatypes would be a powerful ratification of species limits. This would be a particularly useful confirmation in scenarios where the putative taxa show only weak morphological and ecological variation. In addition, the differing timescales resolved by Radseq and sequence capture would mean that the delimitation problems are essentially approached from both sides of the species continuum, helping to resolve the boundaries of incipient species, where the lineages are in the 'grey zone' of speciation [@Dequeoz2007; @refs]. Finally, sequencing the same individuals in each dataset would also account for intraspecific variability and allow for direct comparisons between the datasets. 

The grass genus _Ehrharta_ Thunb. currently consists of approximately 45 taxa (species and subspecies) and has a Gondwanan distribution, with species occurring in Australasia, New Zealand, Malesia, Reunion, Madagascar and Africa. However, a substantial proportion of _Ehrharta_ species (22 in total), are endemic to the Greater Cape Floristic Region and comprise of a core low-elevation clade that is subtended by a grade of mid- to high-elevation lineages [@GibbsRussell1987; @Verboom2003]. There is little taxonomic confusion within the low-elevation clade, as the species are clearly morphologically distinct, with characters that easily distinguish the different species. Conversely, within three of the higher-elevation, fynbos-restricted complexes, (called here Setacea, Rehmannii and Ramosa groups), the species limits are less clear. The Setacea group consists of two species, _E. setacea_ and _E. rupestris_, the former comprising four subspecies (_disticha, setacea, uniflora_ and _scabra_), while the latter contains three (_rupestris, tricostata_ and _dodii_). These species and subspecies are morphologically similar and difficult to distinguish [@GibbsRussell1987], although field observations suggest that there may be subtle ecological differentiation. While the subspecies in the Setacea group tend to have distinct ranges, there are areas of overlap where intermediates also occur. Species limits are even less clear within the Ramosa and Rehmannii groups, which each contain a single species: _E. ramosa_ with two subspecies (_aphylla_ and _ramosa_), and _E. rehmannii_ with three subspecies (_rehmannii, filiformis_ and _subspicata_). The most recent infrageneric taxonomy of the Cape _Ehrharta_ states that these "five taxa ... intergrade and few consistently reliable characters adequately separate the species and subspecies" [@GibbsRussell1989]. As the revision of @GibbsRussell1989 used only morphological and anatomical characters, it is possible that the application of genetic data may bring clarity to species boundaries in this group.

Here, we ask whether two independent, multi-locus  data sets, each analysed using different methodological approaches, are consilient in terms of the species-level groupings they define. The high- and mid-elevation Setacea, Rehmannii and Ramosa groups of _Ehrharta_ are strong Moreover, they contain multiple taxonomic entities which, by virtue of their weak morphological differentiation and relatively distinct geographical distribution, are currently conceptualised as subspecies, but may potentially represent distinct species. We conduct comprehensive, population-level sampling across the entire range of the Ramosa, Rehmannii and Setacea groups, with individuals sequenced via both sequence capture and genotyping-by-sequencing. In addition to insights obtained from these data, we employ the variation in morphological traits to assess the evolutionary distinctiveness of putative species within these groups.



With regard to paragraph 4, I find the aims a bit wishy-washy/repetitive as stated. Perhaps our central aims are to assess (i) whether two independent genetic data sets, analysed using different approaches, are consilient in terms of the species-level groupings they define, and (ii) whether these species are morphologically cohesive.

The archipelagic character of the Cape high-elevation zone lends itself to non-ecological vicariant or peripatric speciation, with the consequence that this zone potentially harbours many undiscovered cryptic species [@Britton2014; @Verboom2015]. The high- and mid-elevation Setacea and Ramosa groups of _Ehrharta_ are strong candidates for cryptic speciation, since their distributions encompass multiple mountain ranges between which the presence of deep valleys may restrict gene flow. Moreover, they contain multiple taxonomic entities which, by virtue of their weak morphological differentiation and relatively distinct geographical distribution, are currently conceptualised as subspecies, but may potentially represent distinct species. In this chapter, therefore, I employ an integrative taxonomic approach [@Dayrat2005] to test the hypothesis that subspecies within the mid- to high-elevation Ramosa and Setacea groups represent cryptic species that constitute distinct evolutionary lineages. In order to evaluate the monophyly of these groups and so provide context for their segregation into species, I first present a multilocus phylogenetic tree of the Cape _Ehrharta_ based on genome-wide targeted enrichment data [@Lemmon2012], with population-level sampling of the Setacea and Ramosa groups. In addition to insights obtained from these data, I employ the variation in single nucleotide polymorphisms (SNPs) generated through genotyping-by-sequencing [@Elshire2011], and in morphological traits to assess the evolutionary distinctiveness of putative species within these groups.



In contrast, few studies have attempted to ratify species boundaries by integrating both sequence capture and Radseq datatypes [exceptions being @Linck]. Yet, under certain circumstances, using a 'two-pronged' approach could be strongly advantageous. For example, current methods still find delimiting recent species and lineages within the grey zone of speciation difficult, with sequence capture potentially not having good enough resolution, and Radseq data often leading to oversplitting. The differing information content of Radseq and sequence capture means that the problem is essentially approached from both sides of the species continuum, potentially leading to more robust species limits. 



Grey zone - 

Approaching species delimitation from two different timescales could help resolve species boundaries where their lineages are in the 'grey zone' of speciation. Ambiguous lineages  In addition, Radseq has a tendency to over split species with strong pop structure.Finally, by sequencing the same individuals in both account for intraspecific variability.  People have been trying to do this using SNPs from UCE, but this is just repackaging the same information rather than been different independent measures. Phylogenetics is starting to make use of both types to resolve shallow radiations, yet this trend has yet to catch on in species delimitation studies. Potential reasons for this is expense, and difficulty of reconciling different resolutions. 



The two datatypes have very different phylogenetic resolutions. Radseq assumed to be neutral, reflect population-level processes. Sequence capture in conserved and generally reflects older timescales (with the exception of flanking regions) Both data types therefore have different strengths and weaknesses for delimiting species [as covered extensively in @Harvey]. In brief:
Phylogenetics is starting to make use of both types to resolve shallow radiations, yet this trend has yet to catch on in species delimitation studies. Using both to incoporate both shallow and deep time scales may help resolve persistent delimitation problems, such as grey zone and strong pop structure.



The utility of the datasets depends on the system being studied. Sequence capture data is better at resolving  older timescales, although with flanking regions)



Benefits of intergrating both: Independent - better than trend of calling SNPs from UCE



In order to circumscribe and validate species boundaries, the focus to date has been on the development of numerous methodological approaches. These techniques vary widely, from population genetic clustering methods [@Pritchard2000; @Frichot2014] to phylogenetic methods [@astral], to species-tree based methods [@bpp, @Snapp]. Given a strong emphasis on using as many methods as possible to validate species boundaries [@Carstens2013], it is perhaps unsurprising that researchers have been pushing the limits as to what their datasets can do. For example, Radseq data are now routinely used for phylogenetic analysis. While this has been successful in resolving recent radiations, Radseq data are potentially less informative than sequence capture if there are deep time splits [@ref], and the recovered relationships tend to be sensitive to missing data and bioinformatic processing decisions [@Leache2015]. Another recent trend is to call single nucleotide polymorphisms (SNPs) from sequence capture data in order to utilise population genetic methods [@Zarza2016; @Winker2018; @Chan2020; @Erikson2020]. However, as the resultant SNPs are retrieved from predominantly functional regions, they may not meet the central assumptions of Hardy-Weinberg equilibrium and linkage disequilibrium required by many population-level analyses without rigorous filtering [@Pritchard2000; @ref], in addition to potentially altering how the resulting outputs should be interpreted [@Helyar2011; @Kirk2011; Silva2020].